src/main/java/com/google/sps/data/TimeRange.java

```diff
@@ -1,143 +0,0 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package main.java.com.google.sps.data;
-
-import java.time.Instant;
-import java.time.Duration;
-import java.util.Comparator;
-
-/**
- * Class representing a span of time, enforcing properties (e.g. start comes before end) and
- * providing methods to make ranges easier to work with (e.g. {@code overlaps}).
- */
-public final class TimeRange {
```

Comment:
This is being deleted?

Reply:
l made some small changes in it so it could compile but it wasn't the focus of this PR and Grant would've made the same changes in his other PR. 
lt's not deleted, just not tracked by git anymore. l did `git rm --cached ...`

Reply:
Sure looks deleted to me.

I think this file was present in master, and you're deleting it in this PR. You need to get the original file back.